Controls.js

```diff
@@ -0,0 +1,8 @@
+//there are several joysticks already made open source on github
+//top result: https://github.com/98oktay/react-native-axis-pad
+
+//this looks interesting for the attacker placing/throwing projectiles:
+//https://reactnative.dev/docs/panresponder 
+//https://medium.com/yapsody-engineering/hooks-and-function-components-in-react-native-d4e667c90cda
+
```

Comment:
It's a little weird to have a non-functional file checked into a repository.
One option is run: `git rm Controls.js`
This removes the file *from git tracking*, i.e. on the next commit it will disappear from github.com/nolan-3/Dodgeball, but leave the actual file on your computer (which you can delete or not, if it contains helpful notes).
Later, if you need the file again, you run `git add Controls.js` to begin tracking changes again.